src/main/java/uk/ac/ebi/ena/readtools/sampler/SplitReader.java

```diff
@@ -18,8 +18,8 @@
 import java.util.zip.GZIPInputStream;
 
 import uk.ac.ebi.ena.readtools.loader.common.QualityNormalizer;
-import uk.ac.ebi.ena.readtools.loader.common.feeder.AbstractDataFeeder;
-import uk.ac.ebi.ena.readtools.loader.common.feeder.DataFeederException;
+import uk.ac.ebi.ena.readtools.loader.common.feeder.AbstractDataProducer;
+import uk.ac.ebi.ena.readtools.loader.common.feeder.DataProducerException;
 import uk.ac.ebi.ena.readtools.loader.fastq.DataSpot;
 import uk.ac.ebi.ena.readtools.loader.fastq.DataSpot.DataSpotParams;
 

```

Comment:
Remove SplitReader. I don't see any use for it. Called only from uk.ac.ebi.ena.readtools.sampler.main().

Reply:
Are we sure that no other component is using uk.ac.ebi.ena.readtools.sampler.Main as either a dependency or an executable?

Reply:
Yes, if Eugene is not using it (need to double check with him).